src/codec/mod.rs

```diff
@@ -408,11 +406,8 @@ impl TryFrom<u8> for Exception {
             0x08 => MemoryParityError,
             0x0A => GatewayPathUnavailable,
             0x0B => GatewayTargetDevice,
-            _ => {
-                return Err(Error::new(ErrorKind::InvalidData, "Invalid exception code"));
-            }
-        };
-        Ok(ex)
+            custom => Custom(custom),
```

Comment:
```suggestion
            other => Custom(other),
```

Reply:
Sorry for the confusion. The naming `other` in the match statement was semantically correct here.

Reply:
Nice one, for sure.